amoro-ams/src/main/java/org/apache/amoro/server/table/DefaultTableService.java

```diff
@@ -471,6 +471,11 @@ public TableRuntime getRuntime(Long tableId) {
     return tableRuntimeMap.get(tableId);
   }
 
+  public int getRuntimeCount() {
```

Comment:
What if we have filters?
For example, database is specified in search

Reply:
@huyuanfeng2018 thanks for pointing this out, fix it.